sigs.yaml

```
@@ -1975,6 +1975,11 @@ sigs:
       github: tpepper
       name: Tim Pepper
   subprojects:
+  - name: ci-testing
```

Should we add

```suggestion
  - name: ci-testing
    contact:
      mailing_list: https://groups.google.com/g/kubernetes-sig-node-test-failures
```

?

---

I'm not sure if that really counts as a contact mailing list? more... alerts for interested people that should preferably go in other documentation?

---

We also sometimes send emails to it. It's just overwhelmed by test alerts right now.

---

Ok cool 👍  - I'll add it then (can't easily commit from GH because needing to regenerate the readme too)

---

added